org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/namedobjects/NamedObjectInheritanceTest.xtend

```diff
@@ -19,7 +19,7 @@ class NamedObjectInheritanceTest extends AbstractWollokInterpreterTestCase {
 			
 			program p {
 				assert.equals("abc", myObject.something())
-				assert.equals(myObject, (myObject->"42").getX())
+				assert.equals(true, (myObject == myObject))
```

Comment:
why was this changed ?
In this case the arrow is a message all objects understand that is currently used to create a pair/tuple.

We have a couple of special operators (in terms of the parser) that ends up being regular object messages.